pcsx2/Frontend/GameList.cpp

```diff
@@ -39,7 +39,7 @@
 enum : u32
 {
 	GAME_LIST_CACHE_SIGNATURE = 0x45434C47,
-	GAME_LIST_CACHE_VERSION = 31
+	GAME_LIST_CACHE_VERSION = 2
```

Comment:
```suggestion
	GAME_LIST_CACHE_VERSION = 32
```

Reply:
No, the cache hasn't been updated 32 times. It was accidentally ported from duck station.

Reply:
I just find it awkward having our cache version start ~30 is all. If you don't care that's fine by me.